module/extension/FileWatcher/src/FileWatcher.hpp

```diff
@@ -23,6 +23,20 @@
 
 namespace module::extension {
 
+
+    ///@brief Call the cleanup function and delete the ptr
+    ///@tparam must be be reinterpret castable into uv_handle_t
```

Comment:
```suggestion
    /// @tparam must be be reinterpret castable into uv_handle_t
```
The one above as well. Always reviewing the important things 😂😂